### DISCUSSION TITLE
Include/Ecxclude Topics from poll

### DISCUSSION THREAD

**Discussion by hibbeldi** [zivillian/ism7mqtt#151] [Q&A]
Hi,
thanks for coding this tool!

I was wondering whether it's (already) possible to just include a bunch of particular parameters (i.e. Aussentemperatur) and don't start polling for all of the others (my ism7 hangs after a few hours and I don't need all that traffic)?

(something like:
include_parameters:
  - Aussentemperatur 
  - ...
  )

Cheers
Volker

**Comment by framps** [comment]
I had a similar issue: But I wanted to exclude only some parameters. Therefore I wrote a [small bash script](https://github.com/framps/bashScriptCollection/blob/master/findExistingISM7MQTTparameters.sh).

In your case you call it with 
```
./findExistingISM7MQTTparameters.sh außentemperatur
>>> "Außentemperatur": CGB-2 - 180145
>>> "Außentemperatur": BM-2 - 220103
>>> "Korrektur Außentemperatur": BM-2 - 220126
>>> "Gemittelte Außentemperatur": DHK_BM-2 - 340001
>>> "Normaußentemperatur Heizkurve": DHK_BM-2 - 340014
```
and now you can create a copy of your parameter.json and delete all parameters and keep the one with ptid `220103` only. Keep in mind you have to reduce everything carefully and keep a valid json. That's the tricky part. 

If you want to delete only some parameters that's quite easy. Just delete the ptid. If you want to keep only some parameters it's a bit tricky to get a valid reduced json. But finally that will work.

Whether there is a more elegant way ... I don't know of any.
